.changeset/unlucky-plants-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+"barnard59-cube": major
```

Comment:
I know it's technically a breaking change but no has had the chance to use this yet :)

```suggestion
"barnard59-cube": patch
```